.github/workflows/odh-fork-sync.yml

```
@@ -1,15 +1,18 @@
 name: Call sync on OpenDataHub CFO fork sync
 on:
   release:
-    types: [published]
+    types:
+      - published
   workflow_dispatch:
 
 jobs:
   sync-fork:
     runs-on: ubuntu-latest
     steps:
       - name: Sync-fork
-        env:
-          GITHUB_TOKEN: ${{ secrets.CODEFLARE_MACHINE_ACCOUNT_TOKEN }}
         run: |
           gh workflow run sync-fork.yaml -R github.com/opendatahub-io/codeflare-operator -r main
```

Just optional suggestion:
```suggestion
          gh workflow run sync-fork.yaml --repo github.com/opendatahub-io/codeflare-operator --ref main
```
From my perspective that improves readability of the command.